.upstream-tests/test/std/numerics/complex.number/cases.h

```diff
@@ -16,166 +16,200 @@
 #include <cuda/std/complex>
 #include <cuda/std/cassert>
 
-#ifdef __CUDA_ARCH__
```

Comment:
This needs to be brought to attention, __CUDA_ARCH__ globals are not going to be supportable with NVC++. So this change serves a dual purpose in the future. 

Immediately, however, NVRTC cannot initialize the `testcases` constant using the dynamic floating point values from the likes of `CUDART_INF_F`.